### ISSUE TITLE
Device doesn't show up again after a rescan

### ISSUE THREAD

**Issue by MuratUrsavas** [NebraLtd/hm-config#186]
**Reproduction Sequence**
* Enter into an existing Hotspot in Helium Mobile App
* Tap on settings icon (gear)
* Select Pair
* A RockPi hotspot should show up. Select via tapping on it.
* Pairing service menu should show up
* Tap on rescan
* Device should show up again, but it doesn't

**Acceptance Criteria**
* Device has to show up in the last step
* Device can be selected and service menu has to be available for it

**Comment by pritamghanghas** [comment]
@MuratUrsavas it is a little confusing. 4 to 7 is expected AFAIK. If you have connected to the hotspot. "scan again" will not show the hotspot again because last connection has stopped discovery mode in bluez. Pressing the button is the only way to get it back. 

If it is desired we will have to modify application logic to keep triggering discoverable till a min 5 mins have passed.

**Comment by MuratUrsavas** [comment]
@pritamghanghas I don't think this is what's desired. Because a user can enter into WiFi Network settings accidentally and would like to go back and enter into Diagnostics. Since Helium App is disconnecting with the Android back button it's not possible right away. Either a reboot or a discovery timeout waiting are required to regain access to the device over BT.

**Comment by pritamghanghas** [comment]
Button press will allow him to make it discoverable again.

**Comment by MuratUrsavas** [comment]
Nope, it's not right now. You need to wait for the timeout.

**Comment by marvinmarnold** [comment]
@MuratUrsavas is this still an issue?
